packages/gestalt/src/icons/duplicate.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M20.182 9.545c0-.752-.611-1.363-1.364-1.363h-3v-3a1.364 1.364 0 1 0-2.727 0v3h-3a1.364 1.364 0 1 0 0 2.727h3v3a1.364 1.364 0 1 0 2.727 0v-3h3c.753 0 1.364-.61 1.364-1.364M24 2.455v14.181a2.455 2.455 0 0 1-2.455 2.455H7.364a2.455 2.455 0 0 1-2.455-2.455V2.455A2.455 2.455 0 0 1 7.364 0h14.181A2.455 2.455 0 0 1 24 2.455m-6.545 20.181c0 .753-.611 1.364-1.364 1.364H6.818A6.826 6.826 0 0 1 0 17.182V7.909a1.364 1.364 0 1 1 2.727 0v9.273a4.095 4.095 0 0 0 4.091 4.09h9.273c.753 0 1.364.612 1.364 1.364" fill="#8E8E8E"/></svg>
```

Comment:
@pfarejowicz optimized all icons with svgo. Thanks for the hint!

Reply:
Can we still check with the PDS folks? Icons here shouldn’t come from sketch + svgo manually but instead through production requests

Reply:
yes, I pinged PDS